packages/contracts/foundry.toml

```diff
@@ -1,31 +1,100 @@
+################################################################
+#                   PROFILE: DEFAULT (Local)                   #
+################################################################
+
 [profile.default]
+
+# Compilation settings
 src = 'contracts'
 out = 'forge-artifacts'
+script = 'scripts'
 optimizer = true
+# [Kroma: START]
 optimizer_runs = 10_000
-
-allow_paths = ["../../node_modules"]
 remappings = [
   '@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/',
   '@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/',
   '@rari-capital/solmate/=node_modules/@rari-capital/solmate',
   'forge-std/=node_modules/forge-std/src',
   'ds-test/=node_modules/ds-test/src'
 ]
+# [Kroma: END]
 extra_output = ['devdoc', 'userdoc', 'metadata', 'storageLayout']
 bytecode_hash = 'none'
-build_info = true
 build_info_path = 'artifacts/build-info'
+
+# Test / Script Runner Settings
 ffi = true
+# [Kroma: START]
 fuzz_runs = 16
 no_match_contract = 'EchidnaFuzz'
-
+# [Kroma: END]
 fs_permissions = [
-  { 'access'='read-write', 'path'='./.resource-metering.csv' },
+  { access='read-write', path='./.resource-metering.csv' },
+  { access='read-write', path='./snapshots/' },
+  { access='read-write', path='./deployments/' },
+  { access='read', path='./deploy-config/' },
+  { access='read', path='./periphery-deploy-config/' },
+  { access='read', path='./broadcast/' },
+  { access='read', path = './forge-artifacts/' },
+  { access='write', path='./semver-lock.json' },
+  { access='read-write', path='./.testdata/' },
+  { access='read', path='./kout-deployment' }
 ]
+libs = ["node_modules"]
+
+[fuzz]
+runs = 64
+
+[fmt]
+line_length=120
+multiline_func_header='all'
+bracket_spacing=true
+wrap_comments=true
+ignore = ['contracts/vendor/WETH9.sol']
+
+################################################################
+#                         PROFILE: CI                          #
+################################################################
 
 [profile.ci]
-fuzz_runs = 512
+deny_warnings = true
```

Comment:
Do we need this? This seems not to be in Optimism

Reply:
actually I brought it from Optimism `v1.7.2` tag. Do you think it's better to bring the latest version?